tests/unit_tests/udfs/test_model_downloader_udf.py

```diff
@@ -0,0 +1,103 @@
+import pathlib
+from tempfile import TemporaryDirectory
+from exasol_bucketfs_utils_python.localfs_mock_bucketfs_location import \
+    LocalFSMockBucketFSLocation
+from exasol_udf_mock_python.column import Column
+from exasol_udf_mock_python.connection import Connection
+from exasol_udf_mock_python.group import Group
+from exasol_udf_mock_python.mock_exa_environment import MockExaEnvironment
+from exasol_udf_mock_python.mock_meta_data import MockMetaData
+from exasol_udf_mock_python.udf_mock_executor import UDFMockExecutor
+
+MODEL_NAME = "test-model-name"
+BFS_CONN_NAME = "test_bfs_conn_name"
+MODEL_FILE_DATA_MAP = {
+    "model_file1.txt": "Sample data in model_file1.txt",
+    "model_file2.txt": "Sample data in model_file1.txt"}
+TOKENIZER_FILE_DATA_MAP = {
+    "tokenizer_file1.txt": "Sample data in tokenizer_file1.txt",
+    "tokenizer_file2.txt": "Sample data in tokenizer_file1.txt"}
+
+
+def udf_wrapper():
```

Comment:
I would suggest, to add another out of db test, which actually downloads from hugging face, because this way, you don't need to debug in the UDF, if something goes wrong with the transformer lib.